tests/push_health/test_compare.py

```diff
@@ -1,189 +1,82 @@
+import pytest
 import datetime
 
 import responses
 
-from treeherder.model.models import Push
-from treeherder.push_health.compare import get_commit_history, get_response_object
+from treeherder.model.models import Push, Repository
```

Comment:
Since `mozci` does most of this work, we don't need all these test cases anymore.